packages/site/src/pages/ChildBounties/index.js

```diff
@@ -0,0 +1,79 @@
+// @ts-check
+import React, { useEffect, useMemo, useState } from "react";
+
+import Pagination from "../Bounties/Pagination";
+import Nav from "../Bounties/Nav";
+import ChildBountiesTable from "./ChildBountiesTable";
+import { useDispatch, useSelector } from "react-redux";
+import { useChainRoute, useQuery, useLocalStorage } from "../../utils/hooks";
+
+import {
+  fetchChildBounties,
+  childBountyListSelector,
+  loadingSelector,
+} from "../../store/reducers/bountySlice";
+import { chainSelector } from "../../store/reducers/chainSlice";
+import { compatChildBountyData } from "./utils";
+
+const DEFAULT_PAGE_SIZE = 20;
```

Comment:
these consts seem to be used everywhere, plz move it to some common-folder so they can be shared

Reply:
will do this in next refactor.